pkg/controller/namespace/namespace_controller.go

```diff
@@ -153,13 +145,6 @@ func (r *ReconcileNamespace) Reconcile(request reconcile.Request) (reconcile.Res
 		return reconcile.Result{}, err
 	}
 
-	// skip if openpitrix is not enabled
-	if r.openpitrixClient != nil {
```

Comment:
Since openpitrix client is no longer needed in namespace controller, we'd better remove it from recocileNamespace fields list and initializer signature as well.

Reply:
Do you mean this commit? https://github.com/kubesphere/kubesphere/pull/2195/commits/3778abf767d416ea1e680d2c965de51a66cceec8

Reply:
yes, exactly